RELEASING.md

```diff
@@ -0,0 +1,63 @@
+# Releasing Kubeclient
+
+## Versioning
+Kubeclient release versioning follows [SemVer](https://semver.org/).
+At some point in time it is decided to release version x.y.z.
+Prerequisite: A [changelog](CHANGELOG.md) is generated & merged into the release branch.
```

Comment:
I think we always want a CHANGELOG covering all releases on master (and if releasing a y-stream branch, include at least that branch releases in CHANGELOG on that branch too